docs/source/test_network.md

```diff
@@ -261,6 +261,9 @@ the chaincode on the channel specified using the channel flag (or `mychannel`
 if no channel is specified).  If you are deploying a chaincode for the first
 time, the script will install the chaincode dependencies. You can use the
 language flag, `-l`, to install the Go, typescript or javascript versions of the chaincode.
+
+Before installing the Go,typescript or javascript version of the chaincode make sure that 
+the required language compiler is installed on you local enviroment.
```

Comment:
This is true for Go since the package step uses `go`. I'm not sure about typescript, javascript, or java since chaincode is built for the peer using the relevant docker images.
@mbwhite do you know for sure?

Also, if we are going to put this in, it should be mentioned BEFORE the deployCC command. And there is a type at `you`.

Reply:
@mbwhite what are your thoughts on this one?

Reply:
Sorry missed the initial ping.... 

The approach the `network.sh` takes (specifically in `https://github.com/hyperledger/fabric-samples/blob/main/test-network/scripts/deployCC.sh` ) is to pre-compile the code before packaging the chaincode.  Whilst not strictly required, it can be good practice to check for compilation errors prior to actually deploying the chaincode. 

It's a fair comment to say that the user should have the tooling of their language of interest installed.  Though I would hope that if you where interested in Java, you'd have a Java already installed :-)

Though the next question will be what version?  Best to refer to the chaincode libraries for the latest requirements. 

but for the main branch, Java 8 & Gradle and Node 16.